### PR TITLE
Bug 1140322 - [RTL][Wallpaper] Change wallpaper grid view should be mirrored r=djf.

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a0068ddb9e9a)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5ea021ee3e9c)

File: apps/wallpaper/style/wallpaper.css [hunk 1]
```diff
@@ -22,16 +22,20 @@ html, body {
 }
 
 .wallpaper {
-  border-bottom: 0.1rem solid #000000;
-  border-right: 0.1rem solid #000000;
   position: relative;
   overflow: hidden;
   float: left;
   width: 33%;
+  -moz-margin-end: 0.1rem;
```
Comment (punamdahiya) [inline apps/wallpaper/style/wallpaper.css]: Fixes first two columns shows up flushed against each other in RTL mode. Favor using -margin-end over explicitly specifying border-left in RTL mode.

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a9d4617f75a0)

**Pull request merged**